### PR TITLE
Adding .travis.yml plus super minor changes

### PR DESCRIPTION
**Pull request by alejandro-carstens**
Hi there, 

First of all thanks for building this, it is an amazing package. Saw that the repo was missing a .travis.yml and thought about adding it in case you want to set it up with Travis CI. I also took the liberty of making a few minor changes.

Best regards and thanks again for your great work, it is much appreciate it.

Alejandro

### CODE REVIEW

**Comment by devdinu** [comment]
You could also add `lint` and `vet` and run the tests with `-race` flag as part of the ci

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - 1.9.x
+  - tip
+
+install:
+  - go get github.com/gomodule/redigo/redis
+  - go get github.com/stretchr/testify/assert
+  - go get github.com/robfig/cron
+  
+script:
+  - go test -v
```
Comment (shdunning) [inline .travis.yml]: Per comments: 
> ...run the tests with `-race`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+language: go
+
+services:
+  - redis-server
+
+go:
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
```
Comment (shdunning) [inline .travis.yml]: Per comments:
> ...add versions 1.10 and 1.11 to build matrix.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - tip
+
+install:
+  - go get github.com/gomodule/redigo/redis
+  - go get github.com/stretchr/testify/assert
+  - go get github.com/robfig/cron
+
```
Comment (shdunning) [inline .travis.yml]: Per comments, add this block before `script:`
```
before_script:
  - go vet ./...
```

**Pull request closed without merging**